kustomize/README.md

```diff
@@ -0,0 +1,176 @@
+
+## Supported Deployment Variations
+
+| **Variation Name**                                                                                                         | **Description**                                                                                                                                                                                                                                                                                |
+|----------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Cymbal Branding](https://github.com/GoogleCloudPlatform/microservices-demo/blob/main/docs/cymbal-shops.md)                | Changes all Online Boutique-related branding to Google Cloud's fictitious company — Cymbal Shops. The code adds/enables an environment variable `CYMBAL_BRANDING` in the `frontend` service.                                                                                                 |
+| [Google Cloud Operations](https://github.com/GoogleCloudPlatform/microservices-demo/blob/main/docs/gcp-instrumentation.md) | Enables Monitoring (Stats), Tracing, Profiler, and Debugger for various services within Online Boutique. The code removes the existing environment variables (`DISABLE_STATS`, `DISABLE_TRACING`, `DISABLE_PROFILER`, `DISABLE_DEBUGGER`) from appropriate YAML config files.                  |
+| [Memorystore](https://github.com/GoogleCloudPlatform/microservices-demo/blob/main/docs/memorystore.md)                     | The default Online Boutique deployment uses the default in-cluster `redis` database for storing the contents of its shopping cart. The Memorystore deployment variation overrides the default database with its own Memorystore (Redis) database. These changes directly affect `cartservice`. |
+
+## Deployment Instructions
+
+This section outlines the steps to deploy any of the supported variations of Online Boutique. 
+
+### Prerequisites
+
+Before proceeding, you will need:
+1. a Kubernetes cluster, such as a [GKE (Google Kubernetes Engine)](https://cloud.google.com/kubernetes-engine) cluster or a [minikube](https://minikube.sigs.k8s.io/docs/) cluster.
+1. to set your `kubectl` context to the Kuberneter cluster.
+
+Alternatively, if you would like to quickly provision a GKE cluster and any other Google Cloud Platform (GCP) resources required for the variants that you choose to run (e.g., a Memorystore Redis cart), you can use the [Online Boutique's Terraform setup at /terraform](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/terraform).
```

Comment:
This paragraph is not necessary, and perhaps misleading, because the terraform README does not make any mention of additional resources required by variations. Suggest removing.